.vitepress/constants/mocha_versions.js

```diff
@@ -1,9 +1,9 @@
 const mochaVersions = Object.freeze({
-  "app-latest-tag": "v1.11.0",
-  "app-latest-sha": "21b5bc747c8500e4888474df7d828e66c33f332d",
-  "core-latest-tag": "v1.36.1-tm-v0.34.29",
-  "core-latest-sha": "624c43d4484a785ec855f5fb93ea571c1e728fda",
-  "node-latest-tag": "v0.14.1",
-  "node-latest-sha": "6549873516889aebcb787c4e9f2ae3b3258c386a",
+  "app-latest-tag": "v2.0.0",
+  "app-latest-sha": "3554ad6608e4028e9f7d5f479a350ab12dc4e314",
+  "core-latest-tag": "v1.39.0-tm-v0.34.29",
+  "core-latest-sha": "6333e56ff954fae056f628ad0fe459409a0b3fcd",
+  "node-latest-tag": "v0.16.0-rc0",
```

Comment:
I think Josh's intention was that this PR would merge when Mocha upgraded to celestia-app v2 and v0.16.0-rc0. Since Mocha will upgrade to v0.15.0 today, I'll create a separate PR for that.

Reply:
https://github.com/celestiaorg/docs/pull/1670